### PR TITLE
Fix a regression bug in loopless FVA

### PR DESCRIPTION
**Pull request by cdiener**
Used the loopless objective as the target flux. Also there was a missing with block which would not reset the exchange fluxes correctly. Some other small fixes that do not actually affect anything but make the method more explicit.

### CODE REVIEW

**Comment by Midnighter** [comment]
Partially fix #698

File: cobra/flux_analysis/loopless.py [hunk 1]
```diff
@@ -221,11 +221,11 @@ def loopless_fva_iter(model, reaction, solution=False, zero_cutoff=1e-6):
         # almost loopless solution containing only loops including the current
         # reaction. Than remove all of those loops.
         ll_sol = get_solution(model).fluxes
-        bounds = reaction.bounds
         reaction.bounds = (current, current)
         model.slim_optimize()
         almost_ll_sol = get_solution(model).fluxes
-        reaction.bounds = bounds
+        print(ll_sol, almost_ll_sol)
```
Comment (Midnighter) [inline cobra/flux_analysis/loopless.py]: Could you remove this again, please? I suppose it was for debugging. Otherwise maybe introduce a logger here.

Comment (cdiener) [reply cobra/flux_analysis/loopless.py]: Ouch, sorry. fixed now.

**Comment by cdiener** [comment]
Well, I'm the one that broke it in the first place :sweat_smile:

**Pull request merged**